src/patternfly/utilities/Text/examples/Text.md

```diff
@@ -123,19 +229,18 @@ Text utility
 | `.pf-u-color-200`           | `*`        | Sets font-color to Color 200           |
 | `.pf-u-color-300`           | `*`        | Sets font-color to Color 300           |
 | `.pf-u-color-400`           | `*`        | Sets font-color to Color 400           |
-| `.pf-u-icon-color-light`    | `*`        | Sets font-color to Icon Color Light    |
-| `.pf-u-icon-color-dark`     | `*`        | Sets font-color to Icon Color Dark     |
-| `.pf-u-link-color`          | `*`        | Sets font-color to Link Color          |
 | `.pf-u-active-color-100`    | `*`        | Sets font-color to Active Color 100    |
-| `.pf-u-active-color-200`    | `*`        | Sets font-color to Active Color 200    |
-| `.pf-u-active-color-300`    | `*`        | Sets font-color to Active Color 300    |
 | `.pf-u-active-color-400`    | `*`        | Sets font-color to Active Color 400    |
-| `.pf-u-disabled-color-100`  | `*`        | Sets font-color to Disabled Color 100  |
-| `.pf-u-disabled-color-200`  | `*`        | Sets font-color to Disabled Color 200  |
-| `.pf-u-disabled-color-300`  | `*`        | Sets font-color to Disabled Color 200  |
 | `.pf-u-primary-color-100`   | `*`        | Sets font-color to Primary Color 100   |
-| `.pf-u-primary-color-200`   | `*`        | Sets font-color to Primary Color 200   |
-| `.pf-u-secondary-color-100` | `*`        | Sets font-color to Secondary Color 100 |
```

Comment:
primary-200 and secondary-100 are still in the css

https://github.com/patternfly/patternfly/blob/e3962b3df7ada7a975af955dca7ac07146169eff/src/patternfly/utilities/Text/text.scss#L116-L121